creativecloud/scripts/scripts.js

```diff
@@ -43,8 +43,8 @@ async function loadArticlePromo(getMetadata) {
   const promoEl = document.querySelector('main .promotion');
   const promoMeta = getMetadata('promotion');
   if (!(promoEl) && !(promoMeta)) return;
-  const { default: decoratePromo } = await import('../features/article-promotion.js');
-  decoratePromo(promoMeta);
+  const { default: init } = await import('../blocks/promotion/promotion.js');
+  init(promoMeta);
```

Comment:
We are passing the metadata to this init as the block does not exist on the page till the init completes. The init will create the block if metadata exists and if it does then it will call addPromoContent to update the content in the created element.